packages/cli/package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "rimraf lib && tsc --watch --preserveWatchOutput",
     "build": "rimraf lib && tsc",
     "lint": "tsdx lint",
-    "test": "jest || jest",
+    "test": "tsdx test",
```

Comment:
Why did you make this change? (tsdx isn't used for the CLI package)

Reply:
Well, I had a look at all the other packages and saw that all of them use `tsdx test` for testing, only `cli` does not. I changed it to have consistent a output across all packages.

Another reason I was confused is that tests were run twice before - if this was to counter flaky tests, we should do something about those other than rerunning them. Adding more of them down the road will only amplify flakiness ^^

Even if there are tests we cant get stable, we should limit retrying to those.
Maybe [`jest-retry`](https://github.com/bluzi/jest-retries) or [`jest-circus`](https://github.com/facebook/jest/tree/master/packages/jest-circus) could come in handy?

Reply:
Ok we can try rolling with this